com.webex.Webex.yaml

```diff
@@ -0,0 +1,69 @@
+app-id: com.webex.Webex
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+sdk: org.freedesktop.Sdk
+command: webex
+separate-locales: false
+tags:
+  - proprietary
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --filesystem=xdg-run/pipewire-0
+  - --env=QT_QPA_PLATFORM=
```

Comment:
why do you need to set this? if there is only X11, it will fallback to xcb.

Reply:
No good reason. I copied them from the Zoom and Skype manifests with the intention of removing as many as possible without breaking the package once I had it working to test, but so far I can't get it past crashing on start up.